openff/toolkit/typing/engines/smirnoff/parameters.py

```diff
@@ -3706,6 +3712,15 @@ def create_force(self, system, topology, **kwargs):
             atom_matches, topology, [(atom,) for atom in topology.atoms]
         )
 
+    def _apply_switching_function(self, force: "openmm.NonbondedForce"):
+        """Apply a switching function to a NonbondedForce if self.switch_width is nonzero."""
+        from openff.units.openmm import to_openmm
+
```

Comment:
Per our extensive discussion, it's nonsensical to have a switch width if we're using LJPME, since it's not used anywhere. But it's also not clear that it should be an error since the SMIRNOFF spec doesn't explicitly forbid it. Let's make a note here:
```suggestion
        # TODO: Should this raise an error if switch_width is nonzero but we're using LJPME?
```

Reply:
Added comments at L3690 in https://github.com/openforcefield/openff-toolkit/pull/1198/commits/fdf37159c1eaeea703ce885e35810c47e5f25c51